cypress/features/regression/showEditPod.feature

```diff
@@ -0,0 +1,161 @@
+Feature: Show Edit Pod component
+  I want to test Show Edit Pod component properties
+
+  Background: Open ShowEditPod component page
+    Given I open "ShowEditPod" component page
+
+  @positive
+  Scenario: Verify the inner content of the component
+    # When I open "ShowEditPod" component page
```

Comment:
We shouldn't have commented clauses.

Reply:
In this case, we should, because only `Then` step will stay. So it indicates the context made by the background

Reply:
I see.